data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1226 @@
+	source "Tschyss"
+	to offer
+		has "Acquiescence: offered"
+		not "Gegno Genocide Defense: offered"
+	on offer
+		conversation
+			branch "returned"
+				has "Gegno Asteroid Mining Prologue: deferred"
+
+			`You arrive at your destination: a massive space station. Instead of standing directly behind you in your cockpit this time, the Gegno ended up taking a seat next to one of your open bunks, not moving a muscle up until now. Despite the grandiose nature of the Gegno's one and only space station, it seems very old and inactive. As you are escorted around the spacious interior, of which a majority is only a skeletal frame of pipes, bars, and supportive structure, you see a noticeable lack of personnel and running machinery. Some sections aren't even properly pressurized, but thankfully, your spacesuit helps with that. Occasionally, you come across some workers, but they appear very lethargic, or maybe just unmotivated.`
+			`	Eventually, you end up in a large docking bay where more workers and ships are present. Some of the workers are hauling crates of materials back and forth, with others talking amongst each other. The Gegno with you then motions over to a half-cracked display board on one of the walls across the room. With one last meaningful look, the Gegno leaves you to your own devices. You wonder if you'll ever see them again, considering the way they've been treated by the others.`
+			choice
+				`	(Go to the display.)`
+					goto "board"
+				`	(Try to communicate with the other Gegno around you.)`
+			
+			`	You attempt to get the attention of some of the other Gegno present in the facility, but most of them give you strange glares or outright ignore you, until one seemingly annoyed Gegno gestures to the large cracked display.`
+
+			label "board"
+			`	The display, although cracked and beaten up, still appears otherwise functional. It presents a series of graphics in a list that seems to be updating in real-time. On the left side of each graphic are various types of mineable asteroids, and on the right are tally marks, which thankfully you can understand. It appears to be some sort of job list linked with the station, which implies that the Gegno are at least giving you a chance to work with them here, to "do your part", or perhaps "make up for what you've done."`
+			choice
+				`	(Accept one of the jobs.)`
+				`	(Leave without doing anything.)`
+					goto "nothing"
+			
+			label "accept"
+			`	You fiddle with the screen, trying to decipher how to use it. Some of the Gegno gaze at you for doing so, and while they silently judge, you manage to figure out that it's a touch-sensitive interface, though it takes a lot of force for you to get it to work.`
+			`	The graphic that appears the first time you get the device to work is of an Iron asteroid, with five strange-looking tally marks next to it. Mining for them wasn't exactly the first thing you were expecting to do for the Gegno, but it is a start.`
```

Comment:
```suggestion
			`	The graphic that appears the first time you get the device to work is of an iron asteroid, with five strange-looking tally marks next to it. Mining for them wasn't exactly the first thing you were expecting to do for the Gegno, but it is a start.`
```

Do we always refer to asteroids in this way? Otherwise, this might work better:
```suggestion
			`	The graphic that appears the first time you get the device to work is of a hematite asteroid, rich in iron, with five strange-looking tally marks next to it. Mining for them wasn't exactly the first thing you were expecting to do for the Gegno, but it is a start.`
```

Reply:
I think if we're going to reword it to explain what hematite is, it should use active voice.

```suggestion
			`	The first time you get the device to work, you see an image of a hematite asteroid, rich in iron, with five strange-looking tally marks next to it. Mining for them wasn't exactly the first thing you were expecting to do for the Gegno, but it is a start.`
```

Reply:
Do we ever refer to a hematite asteroid in the game? If so, this is fine. If not, it brings unnecessary confusion, even if we know what that is in real life.

Reply:
We do not. But then again, we do not talk much about asteroid mining in general. This could be a good precedent to set, or it could be a bad one. Up to you.